manifests/params.pp

```diff
@@ -145,22 +151,26 @@
         }
       }
 
-      $service_name       = 'elasticsearch'
-      $service_hasrestart = true
-      $service_hasstatus  = true
-      $service_pattern    = $service_name
-      $defaults_location  = '/etc/sysconfig'
-      $pid_dir            = '/var/run/elasticsearch'
     }
     'Debian', 'Ubuntu': {
       $service_name       = 'elasticsearch'
       $service_hasrestart = true
       $service_hasstatus  = true
       $service_pattern    = $service_name
-      $service_providers  = 'init'
       $defaults_location  = '/etc/default'
-      $init_template      = 'elasticsearch.Debian.erb'
-      $pid_dir            = false
+
+      case $::operatingsystemmajrelease {
+        '8': {
```

Comment:
@mfournier could we change this to everything lower then 8 use init and 8 or above use systemd?
Makes it easier to maintain for the future.

Thanks!